### PR TITLE
Rebalancing of scrap suit

### PR DESCRIPTION
**Pull request by smolbird**
This alters the scrap suit to make it craftable at an earlier level, while altered to be less protective. Before it was available at a point in the game where it competes with plate armor (having better protection than plate armor too) yet okay encumbrance and being infinitely easier to craft than proper armor.

By including iron in the material, reducing the material thickness, and bumping up the encumbrance somewhat, the end result is something in between leather armor and the standard iron/leather armor like mail or plated leather in terms of protection, with an encumbrance that makes it worth considering at the point in the game the recipe becomes available. Boiled leather armor and up will beat it, but these are available at higher levels and are more resource-intensive.

Results according according to the [Armor Balance](https://github.com/CleverRaven/Cataclysm-DDA/wiki/Armor-balance) formula:

| Before | After |
| ------ | -----:|
| 0.741 | 0.385 |

Also changed the item's deceptive description. As far as I can determine, the scrap suit has no effect on movement noise buried in the code, and I don't know if it ever DID have it.

### CODE REVIEW

File: data/json/recipes/armor/suit.json [hunk 1]
```diff
@@ -285,12 +285,13 @@
...
+    "difficulty": 3,
     "time": 100000,
     "reversible": true,
     "autolearn": true,
-    "book_learn": [ [ "recipe_melee", 6 ] ],
+    "book_learn": [ [ "recipe_melee", 2 ] ],
     "using": [ [ "cordage", 2 ] ],
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "HAMMER", "level": 1 } ],
     "components": [ [ [ "scrap", 15 ] ] ]
   },
```
Comment (BorkBorkGoesTheCode) [inline data/json/recipes/armor/suit.json]: The weight of scrap required is significantly less than the weight of the scrap suit.

**Comment by BorkBorkGoesTheCode** [comment]
Fixes #19929 point no. 8

File: data/json/recipes/armor/suit.json (hunk 1 above)
Comment (smolbird) [reply data/json/recipes/armor/suit.json]: Could work.  What about the weight of the strings?

**Comment by Night-Pryanik** [comment]
Jenkins rebuild

**Pull request merged**